doc/TEST.md

```diff
@@ -0,0 +1,62 @@
+# Test
+
+## Install Docker
```

Comment:
We should have `## Container environment`, and then a section for Docker and a section for podman.

The package for the podman docker alias is `podman-docker.noarch`

Reply:
Do we need to change the content of `testsuite.sh` to use `Podman` to manage the process, or is the current setup using the command `docker ... ` okay?

Reply:
The `podman-docker` package will just install a symlink for `docker -> podman`, and all commands are the same.

Just its part of the standard repositories, and you don't have to `systemctl enable` it, so the setup is a lot simpler